react/src/components/HeaderMenu/HeaderMenuDeskop.tsx

```diff
@@ -1,4 +1,4 @@
-import { Link, NavLink } from "react-router-dom";
+import { Link, NavLink, useParams, useSearchParams } from "react-router-dom";
```

Comment:
`useSearchParams` is not used here, not in `HeaderMenuMobile` either, it can probably be removed